src/components/tutorial-meta/components/variant-list/index.tsx

```diff
@@ -0,0 +1,59 @@
+// @TODO uncomment when enabling cookies
+// import Cookies from 'js-cookie'
+import Link from 'next/link'
```

Comment:
🙋🏻 Should this be `component/link`?

Reply:
Forgot about this component! I'll update